### PR TITLE
docs/operation/autostart

### PR DESCRIPTION
**Pull request by PtrckM**
**Before submitting this PR, please make sure**:
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/", etc)

**A description of the changes proposed in the pull request**:
- update strategy autostart
- add other docker options

### CODE REVIEW

File: documentation/docs/operation/client.md [hunk 1]
```diff
@@ -43,6 +43,27 @@ docker run -it \
 --mount "type=bind,source=$(pwd)/hummingbot_files/hummingbot_logs,destination=/logs/" \
 coinalpha/hummingbot:latest
 ```
+```bash tab="Example"
+docker run -it \
+-e STRATEGY=discovery \
+-e CONFIG_FILE_NAME=conf_discovery_strategy_1.yml \
+-e WALLET=0x220d3d1CFb549005a360e4980ed50ae13C2C722D \
```
Comment (carlolm) [inline documentation/docs/operation/client.md]: Can you replace the address to: "<INSERT_WALLET_ADDRESS>"

File: documentation/docs/operation/client.md [hunk 2]
```diff
@@ -51,6 +72,14 @@ bin/hummingbot_quickstart.py \
 --wallet ${WALLET} \
 --wallet-password ${WALLET-PASSWORD}
 ```
+```bash tab="Example"
+bin/hummingbot_quickstart.py \
+--strategy discovery \
+--config-file-name conf_discovery_strategy_0.yml \
+--wallet 0x220d3d1CFb549005a360e4980ed50ae13C2C722D \
```
Comment (carlolm) [inline documentation/docs/operation/client.md]: Can you replace the address to: "<INSERT_WALLET_ADDRESS>"

File: documentation/docs/operation/client.md [hunk 3]
```diff
@@ -43,6 +43,27 @@ docker run -it \
 --mount "type=bind,source=$(pwd)/hummingbot_files/hummingbot_logs,destination=/logs/" \
 coinalpha/hummingbot:latest
 ```
+```bash tab="Example"
+docker run -it \
+-e STRATEGY=discovery \
+-e CONFIG_FILE_NAME=conf_discovery_strategy_1.yml \
+-e WALLET=0x220d3d1CFb549005a360e4980ed50ae13C2C722D \
+-e WALLET_PASSWORD=walletpasshere \
```
Comment (carlolm) [inline documentation/docs/operation/client.md]: Can you replace the WALLET_PASSWORD to: "<INSERT_WALLET_PASSWORD>"

File: documentation/docs/operation/client.md [hunk 4]
```diff
@@ -51,6 +72,14 @@ bin/hummingbot_quickstart.py \
 --wallet ${WALLET} \
 --wallet-password ${WALLET-PASSWORD}
 ```
+```bash tab="Example"
+bin/hummingbot_quickstart.py \
+--strategy discovery \
+--config-file-name conf_discovery_strategy_0.yml \
+--wallet 0x220d3d1CFb549005a360e4980ed50ae13C2C722D \
+--wallet-password walletpasshere
```
Comment (carlolm) [inline documentation/docs/operation/client.md]: Can you replace the WALLET_PASSWORD to: "<INSERT_WALLET_PASSWORD>"

**Comment by PtrckM** [comment]
done, thanks

**Pull request merged**